libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SmartSyncLogger.java

```diff
@@ -213,6 +213,16 @@ public static void v(String tag, String msg, Object obj) {
         v(tag, msg + ": " + objStr);
     }
 
+    /**
+     * Sets the log level to be used.
+     *
+     * @param level Log level.
+     */
+    public static void setLogLevel(SalesforceLogger.Level level) {
```

Comment:
We should offer a similar method (or exposed getLogger() publicly) in all the *Logger classes otherwise there is no way for developers to control log levels per component.